### PR TITLE
Add documentation note about quotas plugin options

### LINKED ISSUE #10311: [DOC]: The new quotas API does not allow to set check interval
### Related problem

The default check interval is 60 seconds, which may be too much for some use cases.
The only way to configure this is to use `.spec.kafka.config`, which is undocumented.

### Suggested solution

Add `checkIntervalSec` to the quotas API with related documentation.

### Alternatives

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by fvaleri**
### Type of change

- Enhancement

### Description

Add documentation note about quotas plugin options,

### Checklist

- [x] Update documentation

Closes #10311.

### CODE REVIEW

File: documentation/modules/managing/proc-setting-broker-limits.adoc [hunk 1]
```diff
@@ -64,6 +64,9 @@ spec:
 NOTE: `minAvailableBytesPerVolume` and `minAvailableRatioPerVolume` are mutually exclusive.
 This means that only one of these parameters should be configured.
 
+NOTE: By default, the storage check interval is 60 seconds.
+This is reasonable, but it is possible to change that in `.spec.kafka.config` setting `client.quota.callback.static.storage.check-interval`.
```
Comment (scholzj) [inline documentation/modules/managing/proc-setting-broker-limits.adoc:68]:
```suggestion
NOTE: Additional options can be configured in the `spec.kafka.config` section.
The full list of supported options can be found in <LinkToTheQuotasPluginRepo>.
```

**Review by scholzj — CHANGES_REQUESTED** [review]

File: documentation/modules/managing/proc-setting-broker-limits.adoc (hunk 1 above)
Comment (scholzj) [reply documentation/modules/managing/proc-setting-broker-limits.adoc:68]: Or at least list the other relevant options as well -> the factor fallback and validity duration.

File: documentation/modules/managing/proc-setting-broker-limits.adoc [hunk 2]
```diff
@@ -64,6 +64,9 @@ spec:
 NOTE: `minAvailableBytesPerVolume` and `minAvailableRatioPerVolume` are mutually exclusive.
 This means that only one of these parameters should be configured.
 
+NOTE: Additional options can be configured in the `spec.kafka.config` section.
+The full list of supported options can be found in the https://github.com/fvaleri/kafka-quotas-plugin?tab=readme-ov-file#properties-and-their-defaults[plugin documentation].
```
Comment (scholzj) [inline documentation/modules/managing/proc-setting-broker-limits.adoc:68]:
```suggestion
The full list of supported options can be found in the https://github.com/strimzi/kafka-quotas-plugin?tab=readme-ov-file#properties-and-their-defaults[plugin documentation].
```

Comment (fvaleri) [reply documentation/modules/managing/proc-setting-broker-limits.adoc]: Good catch, thanks!

**Review by scholzj — APPROVED** [review]
Thanks

**Review by ppatierno — APPROVED** [review]

**Comment by fvaleri** [comment]
@PaulRMellor is this fine for you?

**Review by PaulRMellor — APPROVED** [review]
Thanks

File: documentation/modules/managing/proc-setting-broker-limits.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (fvaleri) [reply documentation/modules/managing/proc-setting-broker-limits.adoc]: @PaulRMellor we could put the check-interval example in the note, that was my original proposal, but looks like there is no agreement on that.

**Pull request merged**